static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -173,6 +181,11 @@
     background: $paid-article-subheader;
     border-top-color: $paid-article-brand;
 
+    &:hover,
+    &:focus {
+        background: darken($paid-article-subheader, 7%);
```

Comment:
I'm afraid here I have to stick to the existing values. In the grand scheme of things, there should be a function that outputs the hover shade, yes; something out of our hands unfortunately.